### ISSUE TITLE
TS101 - Failed to install RC 2.22

### ISSUE THREAD

**Issue by dtdionne** [Ralim/IronOS#1809]
Failed to install on TS101

**Comment by ia** [comment]
> Failed to install on TS101

Please, try to use advises from [here](https://github.com/Ralim/IronOS/discussions/1785#discussioncomment-6661204) (it's for `TS100` but valid for `TS101` as well) & [here](https://github.com/Ralim/IronOS/discussions/1775#discussioncomment-6631046).

If it won't help, then more detailed information is needed like a step-by-step description of what you do and what the current result is.

**Comment by dtdionne** [comment]
Apologies, I just dropped the TS101_EN.hex file on the drive and a few min later it reconnected and was .err instead of .rdy.  I don't know how to pull logs from this.

1. Powered over usbc holding A
2. TS101 drive showed in windows
3. Pasted TS101_EN.hex
4. A few min later it reconnected and the filename was TS101_EN.err

Well good grief!  I tried this at least a dozen times, with the exact same file.  Didnt work.  Figured I might as well record it...and it worked.

I'm sorry.
https://youtu.be/AbFp-VMgfCQ?si=_WO6cpEQ9DH5OWmQ
